site/src/components/AvatarData/AvatarData.tsx

```diff
@@ -54,7 +54,7 @@ export const AvatarData: FC<AvatarDataProps> = ({
 						css={{
 							fontSize: 13,
 							color: theme.palette.text.secondary,
-							lineHeight: "150%",
+							lineHeight: 1.5,
```

Comment:
ah the wonderful subtleties of css. so annoying that this actually means something different. 🙃

Reply:
CSS is so silly sometimes